contrib_bots/lib/xkcd.py

```diff
@@ -0,0 +1,139 @@
+from random import randint
+
+import logging
+import requests
+
+XKCD_TEMPLATE_URL = 'http://xkcd.com/%s/info.0.json'
+LATEST_XKCD_URL = 'http://xkcd.com/info.0.json'
+
+
```

Comment:
Thanks for the link.

If you look at the `run.py`, we're not using the double blank lines - I think the code in `contrib_bots` should be written in the same style, so I'd remove them for now.